StRoot/StEvent/StFwdTrack.h

```diff
@@ -0,0 +1,101 @@
+/***************************************************************************
+ *
+ * $Id: StFwdTrack.h,v 2.1 2021/01/11 20:25:37 ullrich Exp $
+ *
+ * Author: jdb, Feb 2022
+ ***************************************************************************
+ *
+ * Description: StFwdTrack stores the Forward tracks built from Fst and Ftt
+ *
+ ***************************************************************************
+ *
+ * $Log: StFwdTrack.h,v $
+ * Revision 2.1  2021/01/11 20:25:37  ullrich
+ * Initial Revision
+ *
+ **************************************************************************/
+#ifndef StFwdTrack_hh
+#define StFwdTrack_hh
+
+#include "Stiostream.h"
+#include "StObject.h"
+#include <vector>
+#include "StThreeVectorD.hh"
+#include "StContainers.h"
+
+namespace genfit {
+    class Track;
+}
+
+class StFcsCluster;
+
+struct StFwdTrackProjection {
```

Comment:
This should be a class, rather than a struct, and should properly protect its data.  Also, should follow STAR naming conventions.  "cov" --> mCov.   XYZ --> mXyz.